src/unity/lib/unity_sframe.cpp

```diff
@@ -1950,6 +1950,188 @@ void unity_sframe::explore(const std::string& path_to_client, const std::string&
       }
     };
 
+    auto getAccordian = [self, &reader, &ew, &column_names, &empty_tz](std::string column_name, size_t index) {
+        index = index - 1;
```

Comment:
Why subtract 1 from index? That seems strange to me.

Reply:
If it's because some parts of our system expect 0-indexing and some parts expect 1-indexing, let's try to settle on 0-indexing everywhere. Otherwise the mismatch is a likely source of bugs in the future as the code changes (we may forget to increment/decrement somewhere).

Reply:
Yeah it'll introduce bugs for sure, so i'll settle on 0 indexing everywhere